src/main/java/org/jboss/pnc/cleaner/logverifier/BuildLogVerifier.java

```diff
@@ -60,6 +60,7 @@ public class BuildLogVerifier {
     private static final String className = BuildLogVerifier.class.getName();
 
     private final Logger logger = LoggerFactory.getLogger(BuildLogVerifier.class);
+    private final Integer DEFAULT_BIFROST_BATCH_SIZE = 1;
```

Comment:
BatchSize with value 1 means that, 1 batch has size 1. Translated to 1 line per DB request or in other way 100000DB requests for 100000 log lines. Could you put it to something like 10000?

Reply:
sure, I was waiting for your feedback especially about this! setting to 10000 then :-)